tests/cases/standalone/common/flow/flow_advance_ttl.result

```diff
@@ -82,6 +82,7 @@ ADMIN FLUSH_FLOW('test_distinct_basic');
 |  FLOW_FLUSHED  |
 +-----------------------------------------+
 
+-- SQLNESS SLEEP 1s
```

Comment:
Why add so many `sleep`? The test case will be slower.

Reply:
> Why add so many `sleep`? The test case will be slower.

was investigating why the sqlness test with posgresql sometimes failed as if wasn't wait for long enough, now it seems it is due to not waiting for long enough after flush flow, but not have a good enough solution now, maybe wait a while after called `flush_flow` in batching engine?